frontend/tailwind.config.js

```diff
@@ -4,8 +4,5 @@ module.exports = {
   theme: {
     extend: {},
   },
-  plugins: [
-    require("@tailwindcss/forms"),
-    require('@tailwindcss/typography')
-  ],
+  plugins: [require("@tailwindcss/forms")],
```

Comment:
Do we no longer need the typography plugin?

Reply:
Didn't seem to be in use by anything and was throwing loads of errors 😅